celerybeatmongo/schedulers.py

```diff
@@ -56,6 +56,9 @@ def next(self):
     def is_due(self):
         if not self._task.enabled:
             return False, 5.0   # 5 second delay for re-enable.
+        if hasattr(self._task, 'max_run_count') and self._task.max_run_count:
```

Comment:
In #19 I added the possibility for one to specify a custom PeriodicTask model, which doesn't necessarily need to be a subclass of `PeriodicTask` as long as it follows the same API.

Since then I've worked with two different scenarios, where instead of `PeriodicTask`, a custom model was used in order to make changes such as switching some properties from mongoengine fields to dynamically computed properties.

I therefore added the `hasattr` check for backwards compatibility. To make sure that if someone is using a custom model and has defined `max_run_count`, it won't break with attribute errors when upgrading celerybeat-mongo.

If that's not an issue or noone else is using custom models, I can happily remove the check.